notes/roman-urdu/77_Surah_Mursalat.md

```diff
@@ -0,0 +1,26 @@
+# Number 77: Surah Mursalat  
+
+Classification: Makki  
+Rukus: 2  
+Ayahs: 50  
+Subject: Qayyamat  
+Related Surah: Surah Naba  
+
+Qayyamat ka naqsha.  
+- Hawaein/farishton ki qasam; k woh waqia (qayyamat) ho kr rhe ga.
+- Faislay ka din (halakat hai us din jutlany walo k liye) yeh ayat 8 baar repeat hoti h.
+
+Aik qoam jati phr dusri aati.
+
+Allah ki qudratein.
+  1. Embryonic stages of man.
+  2. Zameen aor usme pahaar aor chashme.
+
+Jahhanam ki details.
+
+Faisle ka din.
+- jutlany walo k liye halakat
+- reverse side imaan walo k liye;
+- Mutaqqi k liye nemat aor inaam.
+
+Qufr walo k liye muhlat jo sarkashi kr rhe krlo jb tk faisle ka din ajaye; Agr ab b aankhy ni khuli to kab khuly gi.
```

Comment:
Doesn't matter if this is only a line, it still would be better to have a heading above this and divide it into maybe 2 points?